src/main/java/opendcs/dao/AlgorithmDAO.java

```diff
@@ -1,48 +1,17 @@
-/*
- * $Id$
- * 
- * $Log$
- * Revision 1.6  2016/11/19 15:55:51  mmaloney
- * Join PARM records with parent records to bring in implicit VPD filter in CWMS.
- *
- * Revision 1.5  2016/03/24 19:21:07  mmaloney
- * Support for algo scripts.
- *
- * Revision 1.4  2016/01/27 22:12:10  mmaloney
- * When reading the complete list, get the params and props in one go rather than
- * individually.
- *
- * Revision 1.3  2015/12/30 20:39:23  mmaloney
- * Bugfix: listAlgorithms was using the same Statement in nested queries, causing SQL exception.
- *
- * Revision 1.2  2014/07/03 12:53:41  mmaloney
- * debug improvements.
- *
- * Revision 1.1.1.1  2014/05/19 15:28:59  mmaloney
- * OPENDCS 6.0 Initial Checkin
- *
- * This software was written by Cove Software, LLC ("COVE") under contract 
- * to the United States Government. 
- * 
- * No warranty is provided or implied other than specific contractual terms
- * between COVE and the U.S. Government
- * 
- * Copyright 2014 U.S. Army Corps of Engineers, Hydrologic Engineering Center.
```

Comment:
While this copyright is incorrect, feels strange removing these and not replacing them with anything. As a rule, should I be putting copyright and Apache license statements on the files I'm altering?

Reply:
Now that you mention it, yeah. And I should be too.